tests/test_eigsolver.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+import numpy as np
+from skcosmo.utils import eig_solver
+
+
+class EigSolverTest(unittest.TestCase):
+    def setUp(self):
+        self.X = np.random.uniform(-5, 5, size=(100, 100))
+        self.X -= self.X.mean(axis=0)
```

Comment:
This is centering/scaling X, right? Using the utility function/classs we have for this would make the intent clearer.

Reply:
Done!